data/snippets.conf

```diff
@@ -19,6 +19,13 @@
 
 # Default is used for all filetypes and keys can be overwritten by [filetype] sections
 [Default]
+if=if (%cursor%)%block_cursor%
+else=else%block_cursor%
+for=for (i = 0; i < %cursor%; i++)%block_cursor%
+while=while (%cursor%)%block_cursor%
+do=do\n{\n\t%cursor%\n} while (%cursor%)\n%cursor%
+switch=switch (%cursor%)%brace_open%case %cursor%:\n\t\t%cursor%\n\t\tbreak;\n\tdefault:\n\t\t%cursor%\n%brace_close%%cursor%
+try=try%block%\ncatch (%cursor%)%block_cursor%
```

Comment:
I think we already [1] ruled out setting snippets in `Default` by default, because it might be very annoying if they do something completely wrong for a particular language (think any language that don't have brace syntax).

[1] or maybe I think about it right now, not sure